### PR TITLE
[FAI-5988] Upgrade base image to JDK19 to fix vulnerabilities

### PR DESCRIPTION
**Pull request by JoshSauter**
image

https://us-east-1.console.aws.amazon.com/ecr/repositories/private/385658942031/airbyte/_/image/sha256:28ec7d67c3aa47761fbca09240ef8eafc6ca9815d25c35be02c5cc059fde6ce4/scan-results?region=us-east-1

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -231,16 +231,18 @@ subprojects { subproj ->
...
             def isArm64 = arch == "aarch64" || arch == "arm64"
 
             def buildPlatform = System.getenv('DOCKER_BUILD_PLATFORM') ?: isArm64 ? 'linux/arm64' : 'linux/amd64'
             def alpineImage = System.getenv('ALPINE_IMAGE') ?: isArm64 ? 'arm64v8/alpine:3.14' : 'amd64/alpine:3.14'
             def nginxImage = System.getenv('NGINX_IMAGE') ?: isArm64 ? 'arm64v8/nginx:alpine' : 'amd64/nginx:alpine'
-            def openjdkImage = System.getenv('JDK_IMAGE') ?: isArm64 ? "arm64v8/amazoncorretto:17.0.4" : "amd64/amazoncorretto:17.0.4"
+            def openjdkImage = System.getenv('JDK_IMAGE') ?: isArm64 ? "arm64v8/amazoncorretto:19" : "amd64/amazoncorretto:19"
             def buildArch = System.getenv('DOCKER_BUILD_ARCH') ?: isArm64 ? 'arm64' : 'amd64'
+            
+            project.logger.lifecycle('jdkVersion is: {} for project {}', jdkVersion, subproj.name)
```
Comment (ypc-faros) [inline build.gradle]: Let's remove the log 😉

**Comment by ypc-faros** [comment]
@cjwooo We could only get rid of the last couple of vulns by upgrading the base image to what Airbyte is using (`amazoncorretto:19`)

**Review by ypc-faros — APPROVED** [review]

**Comment by cjwooo** [comment]
Did the Airbyte unit tests pass?

**Review by cjwooo — APPROVED** [review]

**Comment by JoshSauter** [comment]
This change actually still left vulnerabilities (the image was ECR lying to me mid-scan), closing in favor of different changes to address the vulnerabilities

**Pull request closed without merging**